locales/base.json

```diff
@@ -42,6 +42,7 @@
     "ethichub": "Lend affordable capital to unbanked farmers",
     "mezumo": "Buy tokens over time using a simple strategy called Dollar-Cost Averaging (DCA)",
     "uniswap": "Swap tokens or earn yield by providing liquidity",
-    "pixpress": "Swap any amount of any crypto assets with others peer-to-peer and earn rewards"
+    "pixpress": "Swap any amount of any crypto assets with others peer-to-peer and earn rewards",
+    "pooltogether": "PoolTogether is a prize savings protocol, enabling you to win by saving"
```

Comment:
Can we change to something that doesn't included that name of the Dapp in the description and matches the tone of other Dapp descriptions? "Save while participating in prize raffles"

Reply:
agreed!